lib/awspec/ext/object.rb

```diff
@@ -0,0 +1,3 @@
+class Object
+  undef :timeout
```

Comment:
Maybe you set `undef :timeout` to [Awspec::Type::Base](https://github.com/k1LoW/awspec/blob/master/lib/awspec/type/base.rb) .

Reply:
@k1LoW 

Thank you! :bowing_man: 
I moved `undef :timeout` to lib/awspec/ext/object.rb.

Pleaes confirm.